crates/ruff_linter/resources/test/fixtures/refurb/FURB192.py

```diff
@@ -0,0 +1,39 @@
+# Errors
+
+sorted(l)[0]
+
+sorted(l)[-1]
+
+sorted(l, reverse=True)[0]
+
+sorted(l, reverse=True)[-1]
+
+sorted(l, reverse=False)[-1]
+
+sorted(l, key=lambda x: x)[0]
+
+sorted(l, key=key_fn, reverse=True)[-1]
+
+sorted(l, key=key_fn)[0]
+
+# Non-errors
+
+sorted(l, reverse=foo())[0]
+
+sorted([1, 2, 3])[0]
```

Comment:
Why is this example not an error? Because the list is already sorted?

Reply:
Oops - it should be. I forgot to match on `ExprList` as well [here](https://github.com/astral-sh/ruff/blob/b73e47c81cac55ada626532fca795a3b7e2e3eb6/crates/ruff_linter/src/rules/refurb/rules/sorted_min_max.rs#L82)

Reply:
I haven't found another example of outputting an `ast::ExprList` into a String, if there is an example to follow that would be great. I'll keep looking